src/gameboy/include/gameboy/gameboy.h

```diff
@@ -45,9 +46,9 @@ class Gameboy {
 	std::unique_ptr<Cartridge> cartridge;
 
 	/**
-	 * Controller instance
```

Comment:
I think Controller instance still works as the comment, if not something more descriptive.

"An Instance to Controller Interface" sounds a bit awkwards.